cmake/Modules/FindComputeCpp.cmake

```diff
@@ -102,13 +101,17 @@ find_program(ComputeCpp_INFO_EXECUTABLE computecpp_info
   NO_SYSTEM_ENVIRONMENT_PATH)
 
 find_library(COMPUTECPP_RUNTIME_LIBRARY
-  NAMES ComputeCpp ComputeCpp_vs2015
+  NAMES ComputeCpp
   HINTS ${computecpp_find_hint}
   PATH_SUFFIXES lib
   DOC "ComputeCpp Runtime Library")
 
+# Found the library, use only single hint from now on
+get_filename_component(computecpp_library_path "${COMPUTECPP_RUNTIME_LIBRARY}" DIRECTORY)
+set(computecpp_find_hint "${computecpp_library_path}/..")
```

Comment:
Can you change this to a get_filename_component ABSOLUTE as well please? Looks a bit cleaner when debugging if it's canonicalised (which IIRC ABSOLUTE should do).

Reply:
Done in https://github.com/codeplaysoftware/computecpp-sdk/pull/279/commits/c96ea6d33a589fa88dd6596357cdde66b5911f09